app/assets/javascripts/angular/controllers/biz_user/biz_user_page_controller.js.erb

```diff
@@ -0,0 +1,34 @@
+app.controller('BizUserCtrl', ['$scope', '$http', '$auth', '$location', function($scope, $http, $auth, $location){
+  
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+  $scope.authuserid = $auth.validateUser().$$state.value.id //the id of the user
+  $scope.addForm  = {};
+  $scope.form     = {};
+ 
+  $http.get(url + '/biz_users/' + $scope.authuserid + '/services').success(function(response){
+    $scope.services = response;
+  })
+
+  $http.get(url + '/biz_users/' + $scope.authuserid + '/sessions').success(function(response){
+    $scope.sessions = response;
+  })
+
+  $scope.form.create = function(){
+    var data = $scope.addForm
+    ;
+
+    $http.post(url + '/services', data).success(function(response){ 
```

Comment:
I think it'd be a great idea to use bootstrap [alerts](http://getbootstrap.com/components/#alerts) where you have all this empty functions... instead of a console.log, you should alert the user that their service is being created